docs/source/howto/export.rst

```diff
@@ -0,0 +1,57 @@
+.. _how-to:export:
+
+************************************
+How to export data for a publication
+************************************
+
+You have performed your calculations with AiiDA, and now you would like to export the AiiDA provenance graph to make your scientific study reproducible.
+
+Since AiiDA keeps track of the provenance of every computed result, this step is easy:
+tell AiiDA the **final results** you would like to be reproducible, and AiiDA will automatically include their entire provenance using the :ref:`topics:provenance:consistency:traversal-rules`.
+
+Exporting individual nodes
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Let's say the key results of your study are contained in three AiiDA nodes with PKs `12`, `123`, `1234`.
+Exporting those results together with their provenance is as easy as:
+
+.. code-block:: console
+
+    $ verdi export create my-calculations.aiida --nodes 12 123 1234
+
+As usual, you can use any identifier (label, PK or UUID) to specify the nodes to be exported.
+
+.. tip:: The default traversal rules make sure to include the complete provenance of any node specified and should be sufficient for most cases.
+    See ``verdi export create -h`` for ways to modify the traversal rules.
+
+.. tip::
+
+    If you want to make sure that the AiiDA export includes everything you intended, you can create a new profile and import it:
+
+    .. code-block:: console
+
+            verdi quicksetup --profile test-export  # create new profile
+            verdi --profile test-export import my-calculations.aiida
+
+            # now use e.g. the QueryBuilder to query the database
+
+Using groups for data exporting
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If the number of results to be exported is large, for example in a high-throughput study, use the ``QueryBuilder`` to add the corresponding nodes to a group ``my-results`` (see :ref:`how-to:data:organize:group`).
+Then export the group:
+
+.. code-block:: console
+
+    $ verdi export create my-calculations.aiida --groups my-results
+
+Publishing AiiDA export files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+AiiDA export files can be published on any research data repository, for example `Zenodo`_, the `Open Science Framework`_ or the `Materials Cloud Archive`_.
+When publishing AiiDA export files on the `Materials Cloud Archive`_, you also get an interactive *EXPLORE* section, which allows peers to browse the AiiDA provenance graph directly in the browser.
+
+.. note:: Please remember to use **UUIDs** when pointing your colleagues to data *inside* an AiiDA export file, since UUIDs are guaranteed to be universally unique (while PKs aren't).
```

Comment:
I would move this comment earlier, around line 35, after the paragraph "If you want to make sure that the AiiDA export includes everything you intended"